crmprtd/__init__.py

```diff
@@ -189,11 +195,7 @@ def run_data_pipeline(download_func, normalize_func, download_args,
     Session = sessionmaker(engine)
     sesh = Session()
 
-    observations = [align(sesh, row) for row in rows if row]
-    for ob in observations:
-        print(ob)
-    # insert(observations)
-
-
-def subset_dict(a_dict, keys_wanted):
-    return {key: a_dict[key] for key in keys_wanted if key in a_dict}
+    observations = list(filter(lambda ob: ob is not None,
```

Comment:
I could be wrong, but I think that current practice is to shy away from using `filter` + `lambda`. I think that this can still be expressed with:

```
obs = [align(sesh, row) for row in rows]
obs = [ob for ob in obs if ob]
```